### ISSUE TITLE
Project is created when data errors out

### ISSUE THREAD

**Issue by SteadyCadence** [Cadasta/cadasta-qgis-plugin#195]
This is a little related to https://github.com/Cadasta/cadasta-qgis-plugin/issues/194.  When there were errors in the data the project settings were created but with no data.  

This means that you cannot use the same "project name" when you re-upload data.

**Comment by dimasciput** [comment]
Yes we cannot use the same name, because they are already uploaded. Because all the data check happening in cadasta backend, like the location type and party type. If we want this happening in the plugin, then we need to write a lot of logic from cadasta backend to this plugin.
